nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/core/ElasticSearchEmbeddedNode.java

```diff
@@ -67,6 +67,7 @@ public void start() {
                .put("path.data", config.getDataPath())
                .put("cluster.name", config.getClusterName())
                .put("node.name", config.getNodeName())
+               .put("discovery.type", "single-node")
```

Comment:
if i understand the purpose of the `discovery.type` is to specify that our ES cluster is formed from one Node. Is it correct.

If it is the case, what will be the behaviour if for a given customer he want to use a ES cluster with many nodes.

Reply:
I don't think the customer will use the Embedded mode for production :)

Reply:
Yes, we want a single node cluster when using the embedded Elasticsearch mode.
The embedded mode is not supported by Elasticsearch and only for development purposes when using Nuxeo.
For production, a true Elasticsearch cluster is required which of course can have many nodes.

Reply:
yes ;), my question is about the Test, sorry i didn't mention it  in my comment.

Reply:
We don't want to run unit tests against such a configuration. Just run them against a true Elasticsearch.